src/ChapterList.tsx

```diff
@@ -235,6 +235,16 @@ export default function ChapterList({
   };
 
   let leftMenuItem = [newMenuItem, dropdownMenu];
+  
+  leftMenuItem = leftMenuItem.map((item, idx) => {
+    let className = item.className;
+    if (idx !== leftMenuItem.length - 1) {
+      className = `${className} mr-xs`;
+    }
+
+    return Object.assign({}, item, { className });
```

Comment:
````
return {...item, className}
```

Reply:
This was a mistake as I meant to provide type params to Object.assign to force checking! I knew the spread operator did not properly type check items, and I forgot that Object.assign did not infer. Given that you don't have a type for item, happy to make the change. I always did the other way for type checking but forgot to add the type params!

https://www.typescriptlang.org/play?#code/PTAEHUFMBsGMHsC2lQBd5oBYoCoE8AHSAZVgCcBLA1UABWgEM8BzM+AVwDsATAGiwoBnUENANQAd0gAjQRVSQAUCEmYKsTKGYUAbpGF4OY0BoadYKdJMoL+gzAzIoz3UNEiPOofEVKVqAHSKymAAmkYI7NCuqGqcANag8ABmIjQUXrFOKBJMggBcISGgoAC0oACCbvCwDKgU8JkY7p7ehCTkVDQS2E6gnPCxGcwmZqDSTgzxxWWVoASMFmgYkAAeRJTInN3ymj4d-jSCeNsMq-wuoPaOltigAKoASgAywhK7SbGQZIIz5VWCFzSeCrZagNYbChbHaxUDcCjJZLfSDbExIAgUdxkUBIursJzCFJtXydajBDIKMjJBhLAAaoQAWqAAN6KEolBgAfny-XYiGk3wA3Gz2dJuVdUJROMxhQBfRTuGirPAALx59KZAF4WWIeQBGfjSHkAclqqGNoFlwsUypVoG1AHlpAArSCwVABBiCOTMTgAHg1-A1AD4ABS2-jMuEmzDwH6QC2ygCUwtt9tATtd7s93oovoDjKDhdAIdDzNl-AjOu4MbjggTlpTNtV6ajAXbVfbASjNdAxtj8cTVuCKhwzQ8ZC8iDjzmB7BoXzcZmY7AYzEg-Fg0HUiQYwL0Ii8ACIAKKrBiIBb6I9JbFH8AOc3CABykAkR6CKgdXzI73r-HXI5UEcBRXGkPBQCcacdGGLB9BQBBEGhYRLkXCQ42iEQDA4bEFiYVgOB4IISiAA

Reply:
Good to know!